app/views/whatsrunningwhere/WhatsRunningWherePage.scala.html

```diff
@@ -136,3 +143,9 @@ <h1>What's Running Where? @platform.displayName</h1>
     </li>
   }
 }
+
+@matchesStaging(wrw: WhatsRunningWhere, version: WhatsRunningWhereVersion) = @{
+    version.versionNumber.asString==wrw.versions
```

Comment:
can you not compare VersionNumbers without converting to String?

Reply:
This is a good point, I will try without.